controllers/controllers.js

```diff
@@ -0,0 +1,28 @@
+const {fetchTopics, fetchArticlesById} = require('../models/models'); 
+
+const getTopics = (req,res) => {
+    fetchTopics()
+    .then((topics) => {
+        res.status(200).send(topics)
+    })
+    .catch((err) => {
+        console.log(err)
```

Comment:
Remember to invoke `next(err)` in your catch blocks! Otherwise we'll never get into your error handlers

Reply:
done. Thanks :)